### DISCUSSION TITLE
Unable to authenticate on docker compose up

### DISCUSSION THREAD

**Discussion by dinotheextinct** [felddy/foundryvtt-docker#425] [Q&A]
I am trying to start the container on a cloud server with preinstalled livekit docker container.

sudo docker compose up --detach gives:
`Authenticate | 2022-09-01 08:20:02 | [error] Unable to authenticate: request to https://foundryvtt.com/ failed, reason: read ECONNRESET`

I am kinda clueless what this is supposed to mean.

**Comment by felddy** [comment]
This error is from the Node.js server that runs Foundry in the container.  It means that when it tried make a request to `https://foundryvtt.com` the connection was closed by a TCP reset.  This is in an indication that something between the Node.js server and Foundry's website is blocking the traffic.  

The most likely causes are the network policies and configurations applied in your cloud environment.  Make sure you have allowed traffic on port `443` to flow to `foundryvtt.com` and  allow the associated responses to return.

**Comment by saschazone35** [comment]
Would it be possible to get instructions how to setup this docker container in combination with the livekit docker: https://github.com/livekit/livekit
Livekit is probably the standard A/V Module for docker, but it seems the configuration in combination is not as trivial as I was hoping for.

**Reply by felddy** [reply]
Please open a new [feature proposal issue](https://github.com/felddy/foundryvtt-docker/issues/new?assignees=&labels=&template=feature.md) and I'll take a look at it.
